docs/how-to-guides/creating-grc20.md

```diff
@@ -3,137 +3,142 @@ id: creating-grc20
+- **Internet connection**
 
 ## 1. Importing token package
-For this realm, we'll want to import the `grc20` package as this will include the main functionality of our token factory realm.
+For this realm, we'll want to import the `grc20` package as this will include
+the main functionality of our token factory realm.
 
 [embedmd]:# (../assets/how-to-guides/creating-grc20/mytoken-1.gno go)
 ```go
-package mytoken
-
 import (
 	"std"
+	"strings"
 
 	"gno.land/p/demo/grc/grc20"
+	"gno.land/p/demo/ufmt"
 )
 
 var (
 	mytoken *grc20.AdminToken
-	admin   std.Address = "g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj" // set admin account
+	admin   std.Address
 )
 
-// init is a constructor function that runs only once (at time of deployment)
+// init is called once at time of deployment
 func init() {
-	// provision the token's name, symbol and number of decimals
-	mytoken = grc20.NewAdminToken("Mytoken", "MTKN", 4)
+    // Set deployer of Realm to admin 
+    admin = std.PrevRealm().Addr()
+
+    // Set token name, symbol and number of decimals
+    mytoken = grc20.NewAdminToken("My Token", "TKN", 4)
 
-	// set the total supply
-	mytoken.Mint(admin, 1000000*10000) // @administrator (supply = 1 million)
+    // Mint 1 million tokens to admin
+    mytoken.Mint(admin, 1000000*10000)
 }
+
 ```
 
 In this code preview, we have:
-- Defined a new local variable `mytoken` and assigned that the type of pointer to `grc20.AdminToken`.
-- Defined and set the value of local variable `admin` to point to a specific gno.land address of type `std.Address`.
-- Set the value of `mytoken` (type `*AdminToken`) to equal the result of creating a new token and configuring its name, symbol + decimal representation.
-- Minted 1 million `Mytoken` and set the administrator as the owner of these tokens.
+- Defined a new local variable `mytoken` and assigned that the type of
+pointer to `grc20.AdminToken`,
+- Defined and set the value of local variable `admin` to point to a specific
+address of type `std.Address`,
+- Initialize `mytoken` as a new GRC20 token, and set its name, symbol, and 
+decimal values,
+- Minted 1 million units of `My Token` and to the admin's address.
 
 ## 2. Adding token functionality
 
-The following section will be about introducing Public functions to expose functionality imported from the [grc20 package](https://github.com/gnolang/gno/tree/master/examples/gno.land/p/demo/grc/grc20).
-
-Add this imports :
-
-```go
-import (
-	....
-	"strings"
-
-	"gno.land/p/demo/ufmt"
-	"gno.land/r/demo/users"
-
-	pusers "gno.land/p/demo/users"
-)
-```
+In order to call exported functions from the `grc20` package, we also need to 
+expose them in the Realm. 
 
 [embedmd]:# (../assets/how-to-guides/creating-grc20/mytoken-2.gno go)
 ```go
```

Comment:
It would be great to have comments like the first code block

Reply:
Added comments! [c2d43d5](https://github.com/gnolang/gno/pull/1766/commits/c2d43d5d33dc1229640ae0e172e1a3e03872c67c)